### PR TITLE
Add alumni components

### PR DESCRIPTION
**Pull request by akshatshah21**
### Adds an Alumni Page

1. `AlumniPage` is the main component. It consists of a tabbed layout for different years.
2. `AlumniCard` is a card for one individual alumnus. Fake data for now
3. `MobileMenu` is a general menu utility to replace the tabbed layout on smaller screens

image
image
image

The image mask is a circle right now, but the following are also easily available (or any mask, using [this](https://bennettfeely.com/clippy/) great tool)

image
image

The `AlumniCard` doesn't have links or any interactivity yet. I've kept them as plain white `Card`s for now.

Please let me know how this looks.

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
The layout looks good to me, I guess a 3 column grid would look even better. Tagging @Saif807380 as well for this. Since these are cards, they should have external links and some interactivity. But we'll get to that once we reach a conclusion.

**Comment by Saif807380** [comment]
I agree with @ShubhankarKG. A 3 column grid with slightly larger cards would look even better. No changes apart from this. Great Work!

File: new_client/src/App.js [hunk 1]
```diff
@@ -70,6 +71,7 @@ function App() {
               <ProtectedRoute path="/addevent" component={AddEvent} />
               <ProtectedRoute path="/event/edit/:id" component={AddEvent} />
               <Route path="/resources" component={ResourcePage} />
+              <Route path="/alumni" component={AlumniPage} />
```
Comment (ShubhankarKG) [inline new_client/src/App.js:74]: I know this isn't complete, but we use `lazy` for code splitting now, so maybe in a later commit, we can update this as well.

**Comment by akshatshah21** [comment]
Updates:
1. The cards flip on click:
Alumni Page Desktop
Alumni Mobile
2. AlumniPage is now a lazy component.
3. We can add icons for different kinds of links, but that will require changing the `Alumnus` model so that it can have multiple links, like Github, LinkedIn, et cetera.

**Comment by akshatshah21** [comment]
Updates:
1. Added social icons 
2. Changed the Alumnus Schema
Social Icons

**Comment by ShubhankarKG** [comment]
It seems we are missing the Alumni header link on the Navbar. Please add that as well.

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]
Rest of the stuff looks good! Just a small suggestion. Thanks for this.

**Review by ShubhankarKG — APPROVED** [review]
The changes look good to me. Thanks!

**Pull request merged**